README.md

```diff
@@ -86,7 +86,7 @@ Technical Steering Committees (TSC) Meeting:
 ([Convert to your timezone.](https://www.thetimezoneconverter.com/?t=10%3A00&tz=GMT%2B8&))
 
 Regular Community Meeting:
-- Europe Time: **Wednesdays at 16:00-17:30 Beijing Time** (weekly, starting from Feb. 19th 2020).
+- Europe Time: **Wednesdays at 16:00-17:00 Beijing Time** (weekly, starting from Feb. 19th 2020).
```

Comment:
this actually aligns with calendar and meeting note.

Reply:
I think maybe it's more suitable to change the time in meeting note and I will modify it. The regular meetings always end after 17:00.  : )